index.js

```diff
@@ -99,27 +104,35 @@ exports.kinesisHandler = function (records, context, callback) {
       logger.info({'message': 'Posting...'})
       logger.info({'message': 'Response: ' + response.statusCode})
       if (response.statusCode !== 200) {
-        if (response.statusCode === 401) {
-          // Clear access token so new one will be requested on retried request
-          CACHE['accessToken'] = null
-        }
-
         callback(new Error())
         logger.error({'message': 'POST Error! ', 'response': response})
         return
       }
+      logger.info({'message': 'POST Success'})
+    })
+  }
+
 
-      if (error) {
+  function deleteRecords(record){
+    logger.info({'message': 'Deleting records'})
+    var options = {
+      uri: process.env['MLN_API_URL'] + '/teacher_set',
```

Comment:
Could we move the entire URL out into the config?

Reply:
Yes. Let me check @LucasCioffi PR if the endpoints have been updated.